cuda.spec

```diff
@@ -55,7 +55,7 @@ ar p %{SOURCE0} data.tar.xz | tar xv --xz -C %_builddir/tmp
 # extract the contents from the individual .deb archives into
 # %_builddir/tmp/usr/local/cuda-10.0/...
 for FILE in %_builddir/tmp/var/cuda-repo-10-0-local-%{realversion}/*.deb; do
-  ar p $FILE data.tar.xz | tar xv --xz -C %_builddir/tmp
+  ar p %FILE data.tar.xz | tar xv --xz -C %_builddir/tmp
```

Comment:
@fwyzard , do you really mean %FILE here?

Reply:
no... thanks for catching this !